lib/adapters/stdio/write.js

```diff
@@ -21,26 +22,24 @@ var Write = Juttle.proc.sink.extend({
         this.format = options.format ? options.format : 'jsonl'; // default to jsonl
         this.serializer = serializers.getSerializer(this.format, this.getStdout(), {});
 
-        var self = this;
         this.serializer
-        .on('error', function(err) {
+        .on('error', (err) => {
             // during write no fatal errors
-            self.trigger('warning', err);
+            this.trigger('warning', err);
         });
-    },
+    }
 
-    getStdout: function() {
+    getStdout() {
         return process.stdout;
-    },
+    }
 
-    process: function(points) {
+    write(points) {
         this.serializer.write(points);
-    },
+    }
 
-    eof: function() {
+    eof() {
```

Comment:
Well then we should change the contract. Are we synchronously done here after calling `this.serializer.done()` anyway?